### PR TITLE
[LOOP-3842/3843] Capture trend rate and out-of-range condition in glucose sample

### PR DESCRIPTION
**Pull request by darinkrauss**
- https://tidepool.atlassian.net/browse/LOOP-3842
- https://tidepool.atlassian.net/browse/LOOP-3843
- https://tidepool.atlassian.net/browse/LOOP-970
- https://tidepool.atlassian.net/browse/LOOP-1257
- Add glucose trend rate and glucose condition

### CODE REVIEW

**Comment by darinkrauss** [comment]
LW PR for build: https://github.com/tidepool-org/LoopWorkspace/pull/648

**Review by rickpasetto — APPROVED** [review]
LGTM

File: CGMBLEKit/Glucose+SensorDisplayable.swift [hunk 1]
```diff
@@ -71,3 +71,9 @@ extension Glucose: GlucoseDisplayable {
         return nil
     }
 }
+
+extension Glucose {
+    public var condition: GlucoseCondition? {
+        return nil
```
Comment (ps2) [inline CGMBLEKit/Glucose+SensorDisplayable.swift]: Return high/low for 401/39?

Comment (darinkrauss) [reply CGMBLEKit/Glucose+SensorDisplayable.swift]: Well, we can, but I wasn't sure if the 39/401 limit were the same for all versions of Dexcom. I figured that DIY folks could make that determination. However, if you are telling me that all versions of Dexcom use 39 and 401 as LOW and HIGH indicators and we can do the same thing as in Tidepool Loop then I will update this code accordingly.

Comment (darinkrauss) [reply CGMBLEKit/Glucose+SensorDisplayable.swift]: @ps2 What would you like me to do here?

Comment (darinkrauss) [reply CGMBLEKit/Glucose+SensorDisplayable.swift]: Updated to use 40/400 limits for both `HKQuantity` and `GlucoseCondition`.

**Pull request merged**